### ISSUE TITLE
Conflicting password requirements

### ISSUE THREAD

**Issue by blr2004** [pwm-project/pwm#469]
When changing password, one of the requirements being displayed is "Password is not case sensitive" even though we specified to have at least 1 lowercase and 1 uppercase letter.

We are using this version
 pwmVersion="1.8.0-SNAPSHOT" pwmBuild="38473568"

![pwm](https://user-images.githubusercontent.com/41362512/54342944-a59c1400-4678-11e9-9daf-fb9347deac2a.png)

**Comment by puzich** [comment]
See my solution for this. I think we have the same issue: #467

**Comment by jcombs-pointblue** [comment]
The “Password is not case sensitive” is being determined by the LDAPChai library based on what type of directory it’s talking to and the setting of that directory. You can still require mixed case but that doesn’t make the directories evaluation of that password case sensitive. However, I’m not sure the library gets it right when you do have case sensitivity enabled in eDirectory. It’s a bit complicated because if your populating the legacy NDS password then the password is NOT case sensitive.

**Comment by blr2004** [comment]
Hi puzich,
Our "Password Policy Source" is already set to Local.

While checking the password settings I saw that there is a "Password is Case Sensitive" setting.  I changed it to True (Case Sensitive) ...

![image](https://user-images.githubusercontent.com/41362512/54800731-671cdf80-4c9e-11e9-8998-186f533b7d11.png)


and it is now displaying "Password is case sensitive".

![image](https://user-images.githubusercontent.com/41362512/54800722-5c624a80-4c9e-11e9-83a1-28780bc3a62c.png)


I'm not sure on the other impact of this setting aside from it is now displaying "Password is case sensitive" when changing password.  Because previously it is displaying "Password is not case sensitive" even if it is actually accepting case sensitive password.
